threads task two.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
The program doesn't exit after 5 rounds.

Reply:
why not?

Reply:
Why not?